parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -59,42 +61,46 @@
   private final boolean reuseContainers;
   @Nullable
   private final Integer batchSize;
+  @Nullable
```

Comment:
No need for this annotation. Looks like we should have removed it in the other PR as well.

Reply:
OK.